DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -47,7 +47,8 @@ public void Get_course_content_of_partially_complete_course_should_return_course
                 true,
                 85,
                 90,
-                0
+                0,
+                null

```

Comment:
Probably worth having at least one test with a course which has settings, either by changing which course is used for one of these tests or by adding a new test? 🙂

Reply:
Yeah I agree. I'm going to add that parsing now and put tests for that here 🙂

Reply:
There are no active courses that have a non-null course settings for the fields used by the initial menu. Instead I've added a test that modifies a course to have some interesting settings, and then test that the `GetCourseContent` method gets the correct settings back 🙂